errors.go

```diff
@@ -152,9 +152,10 @@ type BadResponseError struct {
 
 func (e BadResponseError) Error() string {
 	return fmt.Sprintf(
-		"expected 200 response from bayeux server, got %d with status '%s'",
+		"expected 200 response from bayeux server, got %d with status '%s' and body '%s'",
```

Comment:
I wonder if we should update the `'%s'` to just `%q` or if that will break backwards compatible parsing.

Reply:
Actually you probably don't need to update V1 here. I keep meaning to clean up the project so it's just v2.